Luke Smith Portfolio.css

```diff
@@ -0,0 +1,106 @@
+body {
+	margin: auto;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.topnav {
+  overflow: auto;
+  background-color: white;
+  position: fixed;
+}
+
+.topnav a {
+  float: left;
+  color: black;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+}
+
+.topnav a:hover {
+  background-color: #ddd;
+  color: black;
+}
+
+body {
+  background: #555;
+}
+
+
+/*About Me Section*/
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  margin: 0;
+  max-width: 500px;
```

Comment:
Your body's maximum width is set to 500px at the moment. Are you going mobile-first?

I like the design, but at some point I'd add in some logic for it to scale to browser size.

If 500px is as small as the page should get, the syntax is min-width: 500px;